### DISCUSSION TITLE
How to setup the Vanjaro source version in local machine and compile it ?

### DISCUSSION THREAD

**Discussion by malanisourabh** [vanjarosoftware/Vanjaro.Platform#1623] [General]
Hello Support,

https://github.com/vanjarosoftware/Vanjaro.Platform/releases/tag/v1.1.0

We have downloaded the source code from above link. 

We need to setup and the Vanjaro source version on local machine to modify the source code and compile it. So we need to have our installable version after compiling our new changes in the Vanjaro module source version.

Please, advise us how to use the module source version and apply our changes to the module.

Thank you for your time.

Best Regards,
Sourabh Malani

**Comment by meetmandeep** [comment]
A webinar is scheduled for next month for a step by step walkthrough of how to setup the development environment: https://www.vanjaro.com/resources/learn/webinars

In the meantime, here are the basic steps: 

1. Install Vanjaro Platform using the [Vanjaro Installer](https://github.com/vanjarosoftware/Vanjaro.Platform/wiki)
2. Fork Vanjaro Repository
3. Clone your fork in the same directory as Vanjaro Installation
